lib/types/post.ts

```
@@ -0,0 +1,182 @@
+  beneficiaries?: IBeneficiary[];
+  preparePeriodEndsAt?: string;
+  decisionPeriodEndsAt?: string;
+  confirmationPeriodEndsAt?: string;
+  timeline?: IStatusHistoryItem[];
+}
+
+export interface Post {
+  id: "";
+  index: number;
+  title: string;
+  content: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)